### PR TITLE
feat: adding setAborted method MOVEMENT-1231

### PR DESCRIPTION
**Pull request by 27Apoorva**

### CODE REVIEW

File: src/actionlib/SimpleActionServer.js [hunk 1]
```diff
@@ -185,6 +185,29 @@ SimpleActionServer.prototype.sendFeedback = function(feedback2) {
...
 
+/**
+*  Set action state to aborted and return to client
+*/
+
+SimpleActionServer.prototype.setAborted = function(result2) {
+    
+
+    var resultMessage = new Message({
+        status : {goal_id : this.currentGoal.goal_id, status : 4},
```
Comment (mbittarelli6river) [inline src/actionlib/SimpleActionServer.js:196]: what's 4? is there an enum we could use or if not maybe we can just add a comment

File: src/actionlib/SimpleActionServer.js [hunk 2]
```diff
@@ -185,6 +185,29 @@ SimpleActionServer.prototype.sendFeedback = function(feedback2) {
     this.feedbackPublisher.publish(feedbackMessage);
 };
 
+/**
+*  Set action state to aborted and return to client
+*/
+
+SimpleActionServer.prototype.setAborted = function(result2) {
```
Comment (mbittarelli6river) [inline src/actionlib/SimpleActionServer.js:192]: `result2`?

File: src/actionlib/SimpleActionServer.js (hunk 1 above)
Comment (27Apoorva) [reply src/actionlib/SimpleActionServer.js:196]: so according to my understanding, `4` is the code for ABORTED state but I took this code from upstream. so if I add enum I should/need to add to the whole code.

File: src/actionlib/SimpleActionServer.js (hunk 2 above)
Comment (27Apoorva) [reply src/actionlib/SimpleActionServer.js:192]: copied from upstream but can change the name. `result2` is used all over lol.

**Pull request merged**